corehq/apps/userreports/tests/test_app_manager_integration.py

```diff
@@ -62,12 +62,13 @@ def _get_column_property(column):
         default_case_property_datatypes = DEFAULT_CASE_PROPERTY_DATATYPES
         [row] = data_source.get_all_values(sample_doc)
         for result in row:
-            self.assertEqual(sample_doc[_get_column_property(result.column)], result.value)
-            if result.column.id in default_case_property_datatypes:
-                self.assertEqual(
-                    result.column.datatype,
-                    default_case_property_datatypes[result.column.id]
-                )
+            if result.column.id != "repeat_iteration":
```

Comment:
non-blocking - someone's going to be super confused/this will behave weird if they make their own column called `repeat_iteration`. i wonder if we should consider creating a reserved namespace for system ids. (I think the same is also true of `doc_id`)